index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/styles/app.css">
+    <title>Document</title>
+</head>
+<body class="page">
+    <header class="header">
+        <img src="assets/icons/menu.svg" alt="Menu">
+    </header>
+    <section>
+        <div class="headline"> 
+          <h1 class="lines"> Hey there <br>
+            I'm THIERRY <br>
+            I'm a Software Developer</h1>
+            <p class="description"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+            <div class="socials">
+              <img src="assets/icons/github.svg" alt="github">
+              <img src="assets/icons/linkeldn.svg" alt="linkeldn">
+              <img src="assets/icons/twitter.svg" alt="twitter">
+              <img src="assets/icons/angellist.svg" alt="angellist">
+              <img src="assets/icons/medium.svg" alt="medium">
+            </div>
```

Comment:
- [ ] Kindly use `<ul>` and `<li>` to group and list your social icons. This is important for semantic reasons
- [ ] Kindly try to replicate as much as possible.  Use background images and adjust text colors. This is important to make your design appealing.

![image](https://user-images.githubusercontent.com/98453979/227333133-921d1d5a-d75a-4dac-9501-74fdad84d1c8.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/227333543-58e5ca4f-72c3-4665-a7da-36d45723b815.png)